db/migrate/20151007121018_create_records.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecords < ActiveRecord::Migration
+  def change
+    create_table :records do |t|
+      t.string  :crime_name, null: false
+      t.boolean :felony
+      t.boolean :misdemeanor
+      t.boolean  :convicted, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.